tools/python3/phloping.py

```
@@ -172,17 +201,36 @@ def print_root_as_csv(scope_timer_file, n_parts, headers=None, regex=None):
             parser.print_help()
             sys.exit(1)
-    st.print_variance_across(scope_timer_filepath)
+    phst.print_variance_across(scope_timer_filepath)
+
+
+def print_scope_timings(scope_timer_filepath=None, sort_worst_first=True, root_id=None):
+    if scope_timer_filepath is None:  # assume cli
+        args = _cli_args().parse_args()
+        scope_timer_filepath = args.file
+        if not scope_timer_filepath:
+            parser.print_help()
```

Ruff (0.8.2)

Undefined name `parser`

(F821)